CONTRIBUTING.md

```diff
@@ -30,6 +30,9 @@ Before making a contribution, it is important to make sure that the change you w
     5. The Lagom API design rules are the following:
         * Features are forever, always think about whether a new feature really belongs to the core framework or if it should be implemented as a module
         * Code must conform to standard style guidelines and pass all tests
+        * Features and documentation must be provided for both Scala and Java API (unless they only make sense for one of the languages)
+        * Java APIs should go to `com.lightbend.lagom.javadsl.xxxxx` in `xxxxx-javadsl` sbt project (the sbt project might be in a folder named xxxx/javadsl or similar)
+        * Scala APIs should go to `com.lightbend.lagom.javadsl.xxxxx` in `xxxxx-scalaadsl` sbt project (the sbt project might be in a folder named xxxx/scaladsl or similar)
```

Comment:
I just want add a comment. Maybe there's a mistake in this PR, maybe just my mistake.
```diff
-        * Scala APIs should go to `com.lightbend.lagom.javadsl.xxxxx` in `xxxxx-scalaadsl` sbt project (the sbt project might be in a folder named xxxx/scaladsl or similar)
+        * Scala APIs should go to `com.lightbend.lagom.scaladsl.xxxxx` in `xxxxx-scaladsl` sbt project (the sbt project might be in a folder named xxxx/scaladsl or similar)
```